images/spire/configs/latest.agent.apko.yaml

```diff
@@ -22,4 +22,4 @@ entrypoint:
 annotations:
   "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
   "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/spire-agent/
-  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/spire-agent
+  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/spire
```

Comment:
JFYI, we've been screwing this up everywhere, and I have a yellow sticky to audit these.

Reply:
Even better, we should be generating these so we can't mess them up.